backend/app/views/spree/admin/shared/_content_header.html.erb

```diff
@@ -1,23 +1,13 @@
 <% if content_for?(:page_title) || content_for?(:page_actions) %>
   <div id="content-header" data-hook>
-    <div class="container">
-      <div class="sixteen columns">
-        <div class="block-table">
-          <% if content_for?(:page_title) %>
-            <div class="table-cell">
-              <h1 class="page-title <%= yield :page_title_classes %>"><%= yield :page_title %></h1>
-            </div>
-          <% end %>
+    <h1 class="page-title <%= yield :page_title_classes %>"><%= yield :page_title %></h1>
 
-          <% if content_for?(:page_actions) %>
-            <div class="page-actions table-cell toolbar" data-hook="toolbar">
-              <ul class="inline-menu">
-                <%= yield :page_actions %>
-              </ul>
-            </div>
-          <% end %>
-        </div>
+    <% if content_for?(:page_actions) %>
+      <div class="toolbar" data-hook="toolbar">
```

Comment:
@jhawthorn is this a kind of class name that we need to keep around? I'd prefer to remove anything we're not actually using.